apis/gcpfirewall/v1/types.go

```diff
@@ -41,6 +41,11 @@ type GCPFirewall struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// DescriptionHello describes a GCP firewall in a human-readable way. Usually it
```

Comment:
DescriptionHello?

Reply:
Sorry, I must have overlooked it after doing some tests